xrdp/xrdp_listen.c

```diff
@@ -204,7 +206,16 @@ xrdp_listen_get_port_address(char *port, int port_bytes,
                             }
                         }
                     }
-
+#if defined(XRDP_ENABLE_VSOCK)
+                    if (g_strcasecmp(val, "use_vsock") == 0)
+                    {
+                        val = (char *)list_get_item(values, index);
+                        if (g_text2bool(val) == 1)
+                        {
+                            *mode = TRANS_MODE_VSOCK;
```

Comment:
Let's be friendly to users. The message is just for an example.

```C
if (g_text2bool(val) == 1)
{
    log_message(LOG_LEVEL_WARNING, "use_vsock is enabled but not built with vsock option.");
#if defined(XRDP_ENABLE_VSOCK)
    *mode = TRANS_MODE_VSOCK;
#endif
}
```

Reply:
Good feedback. Fixed.